src/backend/src/grpc_server.h

```diff
@@ -3,23 +3,23 @@
 #include <grpc++/server.h>
 #include <memory>
 
-#include "plugins/action/action.h"
+#include "action/action.h"
 #include "action/action_service_impl.h"
-#include "plugins/calibration/calibration.h"
+#include "calibration/calibration.h"
 #include "calibration/calibration_service_impl.h"
-#include "plugins/camera/camera.h"
+#include "camera/camera.h"
 #include "camera/camera_service_impl.h"
 #include "core/core_service_impl.h"
 #include "dronecode_sdk.h"
-#include "plugins/mission/mission.h"
+#include "mission/mission.h"
 #include "mission/mission_service_impl.h"
 #include "telemetry/telemetry_service_impl.h"
 #include "info/info_service_impl.h"
-#include "plugins/gimbal/gimbal.h"
+#include "gimbal/gimbal.h"
 #include "gimbal/gimbal_service_impl.h"
-#include "plugins/param/param.h"
+#include "param/param.h"
 #include "param/param_service_impl.h"
-#include "plugins/offboard/offboard.h"
+#include "offboard/offboard.h"
```

Comment:
I don't think I agree with this change? What was your thought?

Reply:
I guess it "just happened" when I moved files (before we had something like `plugins/offboard/include/offboard/offboard.h` that I removed, and possibly it comes from there).

I can move it back to `plugins/offboard/offboard.h` if you want.

Reply:
Looking into it, I changed it because I did not feel like including all of `${PROJECT_SOURCE_DIR}` ([here](https://github.com/Dronecode/DronecodeSDK/blob/reorganize-deps/src/backend/src/CMakeLists.txt#L54). In order to be able to call `#include "plugins/..."` I need to add the directory containing `plugins` to the `target_include_directories`, which would mean that I would include e.g. `integration_tests/` there, right? What do you think?

Reply:
I can revert back to having the header in e.g. `src/plugins/action/include/plugins/action/action.h`. I just thought to remember that at first you were not really in favor of that.

Your call here, I'm personally not sure what's best.

Reply:
Well actually it's maybe better to have the `include/` folder in order to declare that correctly when exporting the headers. I'll change that.

Reply:
I don't understand. What was wrong with how it was?
It was introduced here and I think made sense: https://github.com/Dronecode/DronecodeSDK/pull/435

Reply:
Sure. I just remembered you did not like it at first and thought it was not necessary anymore, but actually I believe it made more sense.